imports/startup/client/routes/clients.js

```diff
@@ -1,12 +1,23 @@
 import { Clients } from '/imports/api/clients/clients';
 import { PendingClients } from '/imports/api/pendingClients/pendingClients';
 import { RecentClients } from '/imports/api/recent-clients';
-import { AppController } from './controllers';
 import Responses from '/imports/api/responses/responses';
+import FeatureDecisions from '/imports/both/featureDecisions';
 import { ClientsAccessRoles } from '/imports/config/permissions';
 import '/imports/ui/clients/clientNotFound';
 import '/imports/ui/clients/selectSurvey';
+import '/imports/ui/clients/searchClient';
+import '/imports/ui/clients/createClient';
+import '/imports/ui/clients/editClient';
+import '/imports/ui/clients/viewClient';
+import '/imports/ui/clients/viewClientMc211';
+import { AppController } from './controllers';
+
+
+const featureDecisions = FeatureDecisions.createFromMeteorSettings();
```

Comment:
Is it possible to create decisions once on startup in a separate module and then import it directly without the need to recreate?